spring-cloud-starter-stream-source-twitterstream/src/main/java/org/springframework/cloud/stream/app/twitterstream/source/AbstractTwitterInboundChannelAdapter.java

```diff
@@ -227,16 +228,16 @@ private void waitLinearBackoff() {
 		}
 	}
 
-	private void waitRateLimitBackoff() {
+	private void waitRateLimitBackoff(HttpStatusCodeException sce) {
 		int millis = rateLimitBackOff.get();
```

Comment:
Well, that was at a glance decision.
But looking closer to the code I see that we really have there some kind of "exponential interval" according Twitter requirements:
> Back off exponentially for HTTP 420 errors. Start with a 1 minute wait and double each attempt. 

So, there is no reason to expose these options since they are standard and hard-coded according to the target protocol.